crates/test-cluster/src/lib.rs

```diff
@@ -90,23 +92,32 @@ pub struct TestCluster {
     pub swarm: Swarm,
     pub wallet: WalletContext,
     pub fullnode_handle: FullNodeHandle,
+
+    // The following clients might be different from those in fullnode_handle if
+    // the rpc is powered by indexer.
+    pub rpc_client: HttpClient,
+    pub sui_client: SuiClient,
+    pub rpc_url: String,
```

Comment:
do we need to set these on the TestCluster struct, or could we in the `rpc_client`, `sui_client` `rpc_url` fns return indexer's if `Some` otherwise default to fullnode

Reply:
Good idea. Addressed in the latest commit.